hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/operations/Backup.java

```diff
@@ -43,25 +43,36 @@
 
 public final class Backup extends Operation implements BackupOperation, IdentifiedDataSerializable {
 
-    private Data backupOpData;
     private Address originalCaller;
     private long[] replicaVersions;
     private boolean sync;
 
     private Operation backupOp;
+    private Data backupOpData;
```

Comment:
do we need to prepend names with `backup`, like `backupOpData` or `backupOp`, they can be understood from context.